src/DiscordHooks.php

```diff
@@ -241,6 +251,13 @@ public static function onUserGroupsChanged( User $user, array $added, array $rem
 	 * @see https://www.mediawiki.org/wiki/Manual:Hooks/UploadComplete
 	 */
 	public static function onUploadComplete( &$image ) {
+		global $wgDiscordNoBots, $wgUser;
```

Comment:
No reason to use `$wgUser` here when the User object is accessible already (line 262).